torch/csrc/distributed/rpc/functions.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include <torch/csrc/distributed/rpc/FutureMessage.h>
+#include <torch/csrc/distributed/rpc/Message.h>
+#include <torch/csrc/distributed/rpc/RpcAgent.h>
+#include <torch/csrc/distributed/rpc/ScriptCall.h>
+#include <torch/csrc/distributed/rpc/ScriptRet.h>
+
+namespace torch {
+namespace distributed {
+namespace rpc {
+
+void processRequestBlocking(
```

Comment:
Add some documentation to this function and also some documentation to this file mentioning what is the purpose of this file and which methods belong here.